src/Polly.Caching/Polly.Caching.csproj

```
@@ -0,0 +1,43 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net8.0;net6.0;netstandard2.0;net472;net462</TargetFrameworks>
```

As this is brand new, it can be leaner in terms of target frameworks.

```suggestion
    <TargetFrameworks>net8.0;netstandard2.0</TargetFrameworks>
```